conf/modules.config

```
@@ -72,19 +72,24 @@ process {
         ]
     }
 
-    withName: 'BWAMEM2_MEM' {
+    // This BWAMEM2_MEM belongs to the decontamination module
+    withName: 'BWAMEM2_MEM_DECONT' {
         cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
         memory = { check_max( 72.GB * task.attempt, 'memory'  ) }
-        time   = { check_max( 16.h  * task.attempt, 'time'    ) }
+        time   = { check_max( 1.h  * task.attempt,  'time'    ) }
 
         ext.args  = "-M"
-        ext.args2 = [
-            '-f',
-            '12',
-            '-F',
-            '256',
-            '-uS',
-        ].join(' ').trim()
+        ext.args2 = "-f 12 -F 256 -uS"
+    }
+
+    // This BWAMEM2_MEM belongs to the coverage module
+    withName: 'BWAMEM2_MEM_COVERAGE' {
+        cpus   = { check_max( 12    * task.attempt, 'cpus'    ) }
+        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
+        time   = { check_max( 1.h  * task.attempt,  'time'    ) }
+
+        ext.args  = "-M"
+        ext.args2 = "-F 12 -F 256 -uS"
```

```suggestion
        ext.args2 = "-F 268 -uS"
```

I've never tried two -Fs with samtools. I think this should be the same as that combined.

---

Alright. I'll revisit this next week, I want to be sure those flags are applied in the right place